app/controllers/users.js

```diff
@@ -43,3 +43,21 @@ exports.userLogIn = (req, res, next) => {
     )
     .catch(next);
 };
+
+exports.getUsersList = (req, res, next) => {
+  const { limit, page } = req.query;
+  const offset = req.skip;
+  return users
+    .getUsers(limit, offset)
+    .then(usersList => {
+      const itemCount = usersList.count;
+      const pageCount = Math.ceil(usersList.count / req.query.limit);
+      return res.status(200).send({
+        users: usersList.rows,
+        page,
+        pageCount,
+        itemCount
+      });
```

Comment:
What I see here, is that you are mapping the response. So, you can use a mapper here. The method would expect `req.query, usersList` and return the response. _More clean code_